bcarpio.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+for n in bob mary joe
```

Comment:
yo bro! We don't make single letter variables here! Get your shit together noob!